p2p/discovery/table.go

```diff
@@ -10,14 +10,13 @@ import (
 	"sort"
 
 	"github.com/seeleteam/go-seele/common"
-	"github.com/seeleteam/go-seele/log"
 )
 
 const (
-	alpha      = 3  // Kademlia concurrency factor
+	alpha              = 1 // Kademlia concurrency factor
```

Comment:
why do you change Kademlia concurrency factor from 3 to 1?

Reply:
just for test, change it back